spec/index.html

```diff
@@ -525,9 +525,9 @@ <h3>Quoted Literals</h3>
         show:218 show:localName "That Seventies Show"@en-us--ltr .         # literal with a language tag, region subtag, and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
```

Comment:
realign the right-side comments. Similar should be done to the 5 lines above these.
```suggestion
        show:218 show:localName "That Seventies Show"@en-us--ltr .           # literal with a language tag, region subtag, and base direction
        show:218 show:localName 'Cette Série des Années Soixante-dix'@fr .   # literal delimited by single quote
        show:218 show:localName "Cette Série des Années Septante"@fr-be .    # literal with a region subtag
```